acceptance/setup/pre_suite/40_install_deps.rb

```diff
@@ -91,5 +91,20 @@
       # Make sure there isn't a gemrc file, because that could ruin our day.
       on master, "rm -f ~/.gemrc"
     end
+
+    # This works around the fact that puppetlabs-concat 1.2.3 requires a ruby in
+    # the normal path, here we work around this for AIO by just installing one
+    # on the database node that needs it.
+    #
+    # https://github.com/puppetlabs/puppetlabs-concat/blob/1.2.3/files/concatfragments.rb#L1
+    #
+    # This can be removed with concat 2.x onces its re-released, as this doesn't
+    # need an external ruby vm, it uses proper types & providers.
+    if options[:type] == 'aio' && test_config[:os_families][database.name] == :Debian &&
```

Comment:
In other areas I think we use "Debian" rather than :Debian, does that matter here?